@kiva/kv-components/src/vue/KvCarousel.vue

```diff
@@ -48,7 +48,24 @@
 				:disabled="embla && !embla.canScrollPrev()"
 				@click="handleUserInteraction(previousIndex, 'click-left-arrow')"
 			>
+				<span
+					v-if="roundedChevron"
+					class="tw-inline-flex tw-w-4"
+				>
+					<svg
```

Comment:
Not sure why we're diverging from the MDI Icons but if we are doing that intentionally, these svgs should be placed in a global location so they can be used elsewhere rather than embedded in this single component.

Reply:
Yeah, there are not MDI rounded chevrons in MDI Icons and Nathan is asking to add them.
What do you suggest to have them placed in a global location?

Reply:
We probably need to start thinkng about a `kv-icons package in this repo. @dyersituations Have you all discussed any broader changes to icons on the site? We should probably get eng + design together to talk through the plans for this...

Reply:
@mcstover this was a one-off request, we haven't seen any other new icons like this.

Reply:
There are others inline svgs across this repo, just following that path...

Reply:
@mcstover because we don't have a set of icons, I think we might have two approaches here:
1. Allow an inline SVG for this use case
2. Push back on design to use the MDI icon
What are you thoughts on the best path forward?

Reply:
If we have a promise that this won't be used in any other interface elements, we can leave it inline. If however, other designers might start using these rounded chevrons in other places like buttons or accordians, we should extract the svg icons.

Reply:
I'll ask, thanks.